examples/passing/LetPattern.purs

```diff
@@ -0,0 +1,128 @@
+module Main where
+
+import Prelude
+import Partial.Unsafe (unsafePartial)
+import Control.Monad.Eff (Eff)
+import Control.Monad.Eff.Console (CONSOLE, log)
+import Test.Assert (ASSERT, assert')
+
+patternSimple :: Boolean
+patternSimple =
+  let (x) = 25252
```

Comment:
Not significant at all. I just thought it's worth checking if it can match `(expr)` well. Would I remove this?